extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -304,6 +304,14 @@ export class AmpA4A extends AMP.BaseElement {
 
     /** @protected {boolean} */
     this.isRelayoutNeededFlag = false;
+
+    /**
+     * Used as a signal in some of the CSI pings. Canary is shortened to 'ca'
+     * and production to 'pr'. TODO(levitzky) Include other release types
```

Comment:
The format for AMP TODOs is `TODO(@glevitzky, #1234)`. I'm surprised Travis didn't block this.